### PR TITLE
[Playwright] Test framework reorg: centralize different enums files into one file

### PR DESCRIPTION
**Pull request by aweng98**
Having one single enum `dsm/enums` file has following benefits:
- Easier to find enums
- Eliminated duplicates
- Easier to externalize strings with enums

### CODE REVIEW

**Comment by aweng98** [comment]
Verify tests in [CircleCI](https://app.circleci.com/pipelines/github/broadinstitute/ddp-angular/27110/workflows/20f596e3-385a-4f75-bec3-542b7cb7837d) right now.

File: playwright-e2e/dsm/enums.ts [hunk 1]
```diff
@@ -0,0 +1,224 @@
+export enum DataFilter {
```
Comment (Rinchan20) [inline playwright-e2e/dsm/enums.ts:1]: nice all-in-one file!

File: playwright-e2e/tests/dsm/participant/participant-sending-sample-received-creates-germline-consent-addendum.spec.ts [hunk 2]
```diff
@@ -132,14 +130,13 @@ test.describe.serial('Sending SAMPLE_RECEIVED event to DSS', () => {
 
       const customizeViewPanel = participantListPage.filters.customizeViewPanel;
       await customizeViewPanel.open();
-      await customizeViewPanel.selectColumns(
-      `Additional Consent: Learning More About Your DNA with Invitae Columns`,
-      ['GERMLINE_CONSENT_ADDENDUM Survey Created']);
+      await customizeViewPanel.selectColumns(CustomizeView.ADDITIONAL_CONSENT_LEARNING_DNA_WITH_INVITAE,
+        ['GERMLINE_CONSENT_ADDENDUM Survey Created']);
```
Comment (Rinchan20) [inline playwright-e2e/tests/dsm/participant/participant-sending-sample-received-creates-germline-consent-addendum.spec.ts]: should probably be changed to: Label.GERMLINE_CONSENT_ADDENDUM_SURVEY_CREATED

**Review by Rinchan20 — APPROVED** [review]
looks good to me - makes things neater

**Pull request merged**